dev-docs/bidders/tapnative.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: bidder
+title: Tapnative
+description: Prebid Tapnative Bidder Adaptor
+pbjs: true
+biddercode: tapnative
+media_types: banner, native
+tcfeu_supported: false
+usp_supported: true
+coppa_supported: true
+gpp_sids: usstate_all
+schain_supported: false
+safeframes_ok: false
+ortb_blocking_supported: false
+dsa_supported: false
+deals_supported: true
+floors_supported: true
+sidebarType: 1
+---
+
+### Banner Params
+
```

Comment:
You're missing the table header here...

```
{: .table .table-bordered .table-striped }
```

Reply:
we have fixed, can you please merge with the main branch